cms-adapter/src/Rsbc.Dmf.CaseManagement/Model/CaseDetail.cs

```diff
@@ -91,6 +91,7 @@ public class CaseDetail
         public string LastName { get; set; }
 
         public string MiddleName { get; set; }
-    }
 
-}
+        public IEnumerable<MedicalCondition> MedicalConditions { get; set; }
```

Comment:
note to self: very low risk but will still require regression testing the portals